network-api/networkapi/settings.py

```diff
@@ -94,6 +95,8 @@
 
 SITE_ID = 1
 
+ENABLE_WAGTAIL = env.ENABLE_WAGTAIL
```

Comment:
I think this isn't quite right. When I pull this branch I get an error: `https://github.com/mozilla/foundation.mozilla.org/pull/1071AttributeError: 'Env' object has no attribute 'ENABLE_WAGTAIL'`